core/object/object.h

```diff
@@ -854,7 +854,7 @@ class Object {
 	bool property_can_revert(const StringName &p_name) const;
 	Variant property_get_revert(const StringName &p_name) const;
 
-	bool has_method(const StringName &p_method) const;
+	virtual bool has_method(const StringName &p_method) const;
```

Comment:
If we're going to make `has_method()` something that child classes can override, we should also make that possible via GDExtension (so, `GDVIRTUAL1R(bool, _has_method, StringName)` and all that)